presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSplitManager.java

```diff
@@ -89,6 +89,22 @@ public void testGetSplitsByNonIdentityPartitionColumnsWithoutFilterPushdown()
         testGetSplitsByNonIdentityPartitionColumns("get_splits_by_nonidentity_without_filter_pushdown", false);
     }
 
+    @Test
+    public void testGetChangelogWithSingleSnapshot()
+    {
+        // Build an Iceberg table, produce an initial snapshot with parent_id=NULL
+        assertUpdate("CREATE TABLE get_changelog_with_single_snapshot (a integer)");
+        assertUpdate("INSERT INTO get_changelog_with_single_snapshot VALUES 1", 1);
+
+        // Verify there is only one snapshot and that the changelog returns empty
+        assertQuery("SELECT COUNT(*) FROM \"get_changelog_with_single_snapshot$snapshots\" GROUP BY snapshot_id", "VALUES 1");
```

Comment:
```suggestion
        assertQuery("SELECT COUNT(*) FROM \"get_changelog_with_single_snapshot$snapshots\"", "VALUES 1");
```

Is it necessary to use `GROUP BY snapshot_id` here?